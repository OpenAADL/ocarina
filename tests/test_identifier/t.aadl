package plop
public

  thread t__a
  end t__a;

  process p_
  end p_;

end plop;
