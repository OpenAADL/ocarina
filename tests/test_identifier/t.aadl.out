t.aadl:4:10: An indentifier cannot have two consecutive underscores
t.aadl:4:10: An indentifier cannot have two consecutive underscores
t.aadl:4:10: An indentifier cannot have two consecutive underscores
t.aadl:4:03: parsing Component_Type, an identifier is expected, found token '_'
t.aadl:4:10: An indentifier cannot have two consecutive underscores
t.aadl:5:07: An indentifier cannot have two consecutive underscores
t.aadl:7:11: An indentifier cannot end with an underscore
t.aadl:7:11: An indentifier cannot end with an underscore
t.aadl:7:03: parsing Component_Type, an identifier is expected, found keyword 'end'
t.aadl:7:11: An indentifier cannot end with an underscore
t.aadl:8:07: An indentifier cannot end with an underscore
t.aadl:10:09: parsing Package_Specification, Empty packages are not allowed
Cannot parse AADL specifications
